wow/core/src/main/resources/db/auth/V3__account_id_long_to_int.sql

```diff
@@ -0,0 +1,5 @@
+begin;
+
+alter table account alter column id type integer;
```

Comment:
Why don't you want to change V2__account directly? (that would imply to drop the previous table if already created?)

Reply:
I invite you to read about [schema migration](https://en.wikipedia.org/wiki/Schema_migration).

The point here is that V3 can be applied to existing databases running the V2 schema.

Reply:
But, I mean, when developing a first version, I don't really mind dropping my tables just for this small change

Reply:
As long as a change remains unpublished, I agree with you, there's no advantage to using the migration system. In that case, I don't exactly remember, but I must have either pushed V2 and then made V3, or just wanted to try out database migrations (or maybe set an example ?).

In any case, as soon as a schema migration is pushed to the repo (even outside of the master branch, specially if more than one people are likely to work on it), then it's necessary to use the schema migration tool as it will avoid mistakes etc etc